src/SignalR/clients/java/signalr/src/main/java/com/microsoft/signalr/HttpHubConnectionBuilder.java

```diff
@@ -54,6 +55,26 @@ HttpHubConnectionBuilder withHttpClient(HttpClient httpClient) {
         this.httpClient = httpClient;
         return this;
     }
+    
+    /**
+     * Sets Json as the {@link HubProtocol} to be used by the {@link HubConnection}.
+     *
+     * @return This instance of the HttpHubConnectionBuilder.
+     */
+    public HttpHubConnectionBuilder withJson() {
```

Comment:
This is the default, right? No need to add it if we're just going to remove it shortly.